rpc/core/tx.go

```diff
@@ -0,0 +1,35 @@
+package core
+
+import (
+	"fmt"
+
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+func Tx(hash []byte, prove bool) (*ctypes.ResultTx, error) {
+	r, err := txIndexer.Tx(hash)
+	if err != nil {
+		return nil, err
+	}
+
+	if r == nil {
+		return &ctypes.ResultTx{}, fmt.Errorf("Tx (%X) not found", hash)
+	}
+
+	block := blockStore.LoadBlock(int(r.Height))
```

Comment:
Just to make sure, we could have Tx as a field of `TxResult` 

```
txResults[txIndex] = &types.TxResult{uint64(block.Height), uint32(txIndex), **txBytes**,*txResult}
```

but this would mean we're duplicating tx bytes (storing them in 2 places) and we do not want that, correct?